garrysmod/lua/includes/util/color.lua

```diff
@@ -22,51 +78,51 @@ end
 -----------------------------------------------------------]]
 function ColorAlpha( c, a )
 
-	return Color( c.r, c.g, c.b, a )
-	
+	c.a = a

```

Comment:
How so? There's no need to create an entire new object:  this function is completely useless anyway.